CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Change log
 ==========
 
+2.2.1 - 2022-09-15
```

Comment:
Could you change this date to "unreleased"? Thanks

Reply:
sure, done.